dbt/README.md

```diff
@@ -256,13 +257,13 @@ dbt test --select default.vw_pin_universe
 Run only one test:
 
 ```
-dbt test --select vw_pin_universe_unique_by_14_digit_pin_and_year
+dbt test --select default_vw_pin_universe_unique_by_14_digit_pin_and_year
```

Comment:
This test got renamed in https://github.com/ccao-data/data-architecture/pull/236.